rocAL/rocAL_pybind/examples/rocAL_api_coco_pipeline.py

```diff
@@ -287,29 +291,19 @@ def coco_anchors():  # Should be Tensor of floats in ltrb format - input - Mx4 w
             res_images, saturation=saturation, contrast=contrast, brightness=brightness, hue=hue)
         flip_coin = fn.random.coin_flip(probability=0.5)
         bboxes = fn.bb_flip(bboxes, ltrb=True, horizontal=flip_coin)
-        if args.display:
-            cmn_images = fn.crop_mirror_normalize(ct_images,
+        cmn_images = fn.crop_mirror_normalize(ct_images,
```

Comment:
is the cmn output float here?

Reply:
Modified all CMN nodes in python examples to use NHWC/U8 format by default